provider/postgis/postgis.go

```diff
@@ -587,6 +591,10 @@ func (p Provider) TileFeatures(ctx context.Context, layer string, tile provider.
 		}
 	}
 
+	if rows.Err() != nil {
+		return rows.Err()
+	}
+
```

Comment:
I go back and forth on what's best with those situations and I think it's ultimately preference. On one hand you can just return it, on the other hand this check is explicit and additional logic could be added after it.

Reply:
I treat this very much like:
```go

if v == true {
   // ...
} else {
   // ...
} 

```

It's better to make it more straightforward to read:

```go

if v {
   // ...
} else {
  // ...
}
```

With something like this, it takes a bit more time to read and understand what is going on.
Which is we are returning whatever error the pgx is producing and not doing anything else. Now, if later we do add some additional processing it will be changed to a check, and it one would know that there is more to do.